src/common/config_opts.h

```diff
@@ -816,6 +816,8 @@ OPTION(osd_op_thread_suicide_timeout, OPT_INT, 150)
 OPTION(osd_recovery_thread_timeout, OPT_INT, 30)
 OPTION(osd_recovery_thread_suicide_timeout, OPT_INT, 300)
 OPTION(osd_recovery_sleep, OPT_FLOAT, 0.01)         // seconds to sleep between recovery ops
```

Comment:
Actually with your change nobody is left using "osd_recovery_sleep," so shouldn't it be deleted from config options?